project/BuildSettings.scala

```diff
@@ -127,7 +121,7 @@ object BuildSettings {
   lazy val dockerSettings = Seq(
     // Use single entrypoint script for all apps
     Universal / sourceDirectory := new java.io.File((LocalRootProject / baseDirectory).value, "docker"),
-    Docker/ maintainer := "Snowplow Analytics Ltd. <support@snowplowanalytics.com>",
+    Docker    / maintainer := "Snowplow Analytics Ltd. <support@snowplowanalytics.com>",
     dockerBaseImage := "snowplow/base-debian:0.2.2",
```

Comment:
I would like to see us move away from `snowplow/base-debian` and instead use `adoptopenjdk:11-jre-hotspot-focal` as a base image.  Again, I know this is completely unrelated to the current PR.  If you're interested you can take a look at where I did this [for the snowplow postgres loader](https://github.com/snowplow-incubator/snowplow-postgres-loader/blob/aa9a2a3da08f2de6f32eb2cdbdb4b430f3cd5497/project/BuildSettings.scala#L50)

Reply:
https://github.com/snowplow-incubator/snowplow-bigquery-loader/issues/188